backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Linker.kt

```diff
@@ -158,7 +159,8 @@ internal class Linker(val context: Context) {
                     kind = linkerOutput,
                     outputDsymBundle = context.config.outputFiles.symbolicInfoFile,
                     needsProfileLibrary = needsProfileLibrary,
-                    mimallocEnabled = mimallocEnabled)
+                    mimallocEnabled = mimallocEnabled,
+                    sanitizer = SanitizerKind.NONE)
```

Comment:
What if we want to use several sanitizers at the same time?

Reply:
For the sanitizers we want to use: asan, tsan - mixing is not even allowed. If in the future we'd want to combine several other sanitizers, we could either bundle them under one `SanitizerKind` entry, or refactor the code by passing a list of `SanitizerKind`s. For now, I think, a more simple implementation would do.